### PR TITLE
Complete user card #14

### LINKED ISSUE #14: User Story 14, Author Show Page
```
As a Visitor,
When I visit an author's show page
I see all books by that author
Each book should show:
- the book title
- the number of pages in the book
- the year the book was published
- a small image of the book cover

Each book should show a list of any other authors
(exclude this show page's author from that list)
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written
- [ ] edge case code written
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #15

child of #33

depends on #17

### PR DESCRIPTION
**Pull request by davehardy632**
Closes #14

### CODE REVIEW

File: spec/features/authors/show_spec.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      expect(page).to have_content(book_3.pages)
+      expect(page).to have_content("Published in #{book_3.year}")
+      expect(page).to have_css("#book-image-#{book_3.id}")
+
+      within("#book-id-#{book_1.id}") do
+
+        expect(page).to have_content("Michael Crichton")
+        expect(page).to_not have_content("Ken Follet")
+      end
+      it 'books show highest rated review' do
```
Comment (stiehlrod) [inline spec/features/authors/show_spec.rb:41]: @davehardy632 been looking at this code with @iandouglas and he says not to merge our own code and to make sure all tests are passing before making a pull request.

File: spec/features/reviews_user/user_show_spec.rb [hunk 2]
```diff
@@ -6,25 +6,40 @@
...
-      expect(page).to_not have_content("Rating: #{@r2.rating}")
-      expect(page).to have_content(@r1.description)
-      expect(page).to_not have_content(@r2.description)
-      expect(page).to have_content(@r1.created_at)
-      expect(page).to have_content(@leaves.title)
-      expect(page).to have_content(@leaves.cover_image)
-      expect(page).to have_content(@omens.title)
-      expect(page).to have_content(@omens.cover_image)
-    end
+  # describe 'when i visit the user show page' do
```
Comment (stiehlrod) [inline spec/features/reviews_user/user_show_spec.rb:14]: @davehardy632 please remind me why this is commented out
